DESCRIPTION

```diff
@@ -59,7 +59,9 @@ Imports:
     tibble,
     ggplot2,
     jsonlite,
-    glue
+    glue,
+    cards,
```

Comment:
As cards and pharmaverseadam are just needed for the website right now (vignettes aren't in the package build), they don't need to be dependencies of the package. There is a field that can be used just for website deps "Config/Needs/website" - let's use that : https://r-pkgs.org/dependencies-in-practice.html#sec-dependencies-nonstandard-config-needs